src/test/java/com/ddd/moodof/acceptance/TagAcceptanceTest.java

```diff
@@ -0,0 +1,92 @@
+package com.ddd.moodof.acceptance;
+
+import com.ddd.moodof.application.dto.TagDTO;
+import com.ddd.moodof.domain.model.user.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.web.util.UriComponentsBuilder;
+import java.util.List;
+import static com.ddd.moodof.adapter.presentation.TagController.API_TAG;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class TagAcceptanceTest extends AcceptanceTest{
+    private Long userId;
+
+    @Override
+    @BeforeEach
+    void setUp() {
+        super.setUp();
+        User user = signUp();
+        userId = user.getId();
+    }
+
+    @Test
+    public void 유저아이디_태그_전체_조회() throws Exception {
+        // given
+        태그_생성(userId, "tag1");
+        태그_생성(userId, "tag2");
+        태그_생성(userId, "tag3");
+        TagDTO.TagResponse tagResponse = 태그_생성(userId, "tag4");
+
+        // when
+        String uri = UriComponentsBuilder.fromUriString(API_TAG)
+                .build().toUriString();
+        List<TagDTO.TagResponse> response = getListWithLogin(uri, TagDTO.TagResponse.class, userId);
+
+        // then
+        assertAll(
+                () -> assertThat(response.get(0)).isNotNull()
+        );
+    }
+
+    @Test
+    public void 태그를_생성한다() throws Exception {
+        // given
+        TagDTO.TagResponse response = 태그_생성(userId, "tagName1");
+
+        // when
```

Comment:
```suggestion
        // when
        TagDTO.TagResponse response = 태그_생성(userId, "tagName1");
```

맥락상 when이 더 적절한 것 같습니다 🙂 (태그 생성을 할때 ~)

given은 로그인이 되어있다 (BeforeEach에서 진행된)로 봐도 될 것 같아요